person.rb

```diff
@@ -28,4 +31,9 @@ def can_use_services?
 
     false
   end
+
+  def add_rental(rental)
+    rental.person = self
+    @rentals << rental
+  end
```

Comment:
In order to achieve the `has-many` associations, the person itself should be present as a parameter to create the new rental. Kindly consider creating a new rental in this method for adding rentals and pass the date and book  as parameters. It should be similar to something below: 

       def add_rental(book, date)
          Rental.new(date, book, self)
       end